helmfile/values/networkpolicy/service-cluster.yaml.gotmpl

```
@@ -57,6 +57,10 @@ harbor:
 monitoring:
   enabled: {{ .Values.networkPolicies.monitoring.enabled }}
   grafana:
+    externalDataSources:
+      enabled: {{- toYaml .Values.networkPolicies.monitoring.grafana.externalDataSources.enabled | nindent 8 }}
```

```suggestion
      enabled: {{ .Values.networkPolicies.monitoring.grafana.externalDataSources.enabled }}
```
Since this is a simple value you don't need to eat the whitespace in front of it `{{-`, generate it as yaml, and indent it. Makes it a bit simpler.